payjoin/tests/integration.rs

```diff
@@ -82,10 +82,10 @@ mod integration {
             // Inside the Sender:
             // Sender create a funded PSBT (not broadcasted) to address with amount given in the pj_uri
             let uri = Uri::from_str(&pj_uri.to_string())
-                .unwrap()
+                .map_err(|e| e.to_string())?
```

Comment:
Seems like these bitcoin_uri errors should be `std::error::Error` if `std` is available, as `rust-bitcoin` is (they're [really fancy](https://github.com/rust-bitcoin/rust-bitcoin/issues/3317) about this), and then `.map_err(Box::from)` would be appropriate to keep the full error information. But I recognize this is aspirational and extremely detailed. Something to note in `bitcoin_uri` for completeness.